From e02c406eee7054df0ed811927edb312a1fddfb51 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 18 Nov 2021 17:06:20 +0800
Subject: [PATCH 53/64] Revert "drm: xlnx: zynqmp_dpsub: Call
 pm_runtime_get_sync before setting pixel clock"

This reverts commit f76f78f9f479ebfdd94ec8c868b39aba7ca4fb0f.
---
 drivers/gpu/drm/xlnx/zynqmp_disp.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/zynqmp_disp.c b/drivers/gpu/drm/xlnx/zynqmp_disp.c
index 205c72a249b7..8cd8af35cfaa 100644
--- a/drivers/gpu/drm/xlnx/zynqmp_disp.c
+++ b/drivers/gpu/drm/xlnx/zynqmp_disp.c
@@ -1447,10 +1447,9 @@ zynqmp_disp_crtc_atomic_enable(struct drm_crtc *crtc,
 	struct drm_display_mode *adjusted_mode = &crtc->state->adjusted_mode;
 	int ret, vrefresh;
 
-	pm_runtime_get_sync(disp->dev);
-
 	zynqmp_disp_crtc_setup_clock(crtc, adjusted_mode);
 
+	pm_runtime_get_sync(disp->dev);
 	ret = clk_prepare_enable(disp->pclk);
 	if (ret) {
 		dev_err(disp->dev, "failed to enable a pixel clock\n");
-- 
2.27.0

